front-end/src/components/CreateTodo.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import useCreateTodo from "../hooks/useCreateTodo";
+import { TodoType } from "../types";
+
+function CreateTodo() {
+    const addHandler = useCreateTodo();
+    const todoItem: TodoType = {
+        id: 3,
+        title: "test",
+        description: "test",
+        needsReminder: false,
+        createdAt: Date(),
```

Comment:
Should this be `Date()` or `new Date()`?

Reply:
new is just an extra thing. Date() works too